### ISSUE TITLE
save-export-clean should flatten

### ISSUE THREAD

**Issue by akkana** [akkana/gimp-plugins#4]
Several people have commented that save-export-clean saves only the current layer. If there are multiple layers and we're saving to a format other than XCF, we should flatten. Preferably without pestering the user with dialogs, because if we're saving to jpg, of _course_ we want to flatten, duh!

**Comment by HybridDog** [comment]
Some time ago l avoided updating my gimp version from 2.6 to 2.8 mainly because l used gimp to edit textures and it was very useful that l only needed to open one, draw a few lines, hit ctrl+s and close to edit it. But then l switched to 2.8 because G'MIC didn't longer work right l think.
A few days ago l disabled the warning message which told me to save the file before closing. Now l can enable this setting again, thanks to the clean export plug-in.
What is the difference between what ```Saver``` and ```Save/Export clean``` do?

**Comment by akkana** [comment]
The page linked off the README describes the difference between the two.
http://www.shallowsky.com/software/gimp-save/

**Comment by HybridDog** [comment]
So Saver is newer than Save/Export clean?

**Comment by akkana** [comment]
Yes. I thought that was clear from "But I found that I wasn't using Save/export clean much myself, and eventually, I wrote a more general plug-in called Saver." Is there a way I could rephrase that to make it more clear?

**Comment by HybridDog** [comment]
in the README the save-export-clean.py has this in its description ```For people who dislike the Save vs. Export split introduced in 2.8.```, so l didn't know if that also belongs to the saver plug-in, maybe you could write that save-export-clean is an alternative to saver

**Comment by akkana** [comment]
I've changed it to this:

save-export-clean.py
  Save or export, marking the current image clean.
  This was an initial quick hack for people who dislike the Save vs. Export
  split introduced in GIMP 2.8; it's mostly obsoleted now by Saver.

Is that clearer?

**Comment by HybridDog** [comment]
yes, thanks
